app/data/contactAPI.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * The ContactAPI is a JavaScript object that contains an array to hold the contact objects,
+ * and methods for manipulating the contacts array or speciifc objects within the array
+ **/
+
+export const ContactAPI = {
+  contacts: [],
+  all: function () {
+    return this.contacts;
+  },
+  addContact: function ({ id, name, email, phone_number, image_url }) {
+    this.contacts.push({ id, name, email, phone_number, image_url });
+  },
+  remove: function (contact) {
+    const index = this.contacts.findIndex((c) => c.id === contact.id);
```

Comment:
be more clear, this not just the index its:
```suggestion
    const contactIdIndex = this.contacts.findIndex((c) => c.id === contact.id);
```